scu-es/test/fiskaltrust.Middleware.SCU.ES.UnitTest/TicketBaiRequestFactoryTests.cs

```diff
@@ -11,7 +11,7 @@ public class TicketBaiRequestFactoryTests
         [Fact]
         public void CreateSignedXmlContent_Should_Create_SignedXml()
         {
-            var cert = new X509Certificate2(@"TestCertificates/PertsonaFisikoa_PersonaFísica.p12", "IZDesa2021", X509KeyStorageFlags.Exportable | X509KeyStorageFlags.PersistKeySet);
+            var cert = new X509Certificate2(@"TestCertificates/autonomo_act.p12", "Iz3np32023", X509KeyStorageFlags.Exportable | X509KeyStorageFlags.PersistKeySet);
```

Comment:
Should we put this secret into a environment variable or is it okay to have this in for testing purposes?